app/views/notices/shop_employee_notices/13_employee_open_enrollment_reminder.html.erb

```diff
@@ -1,33 +1,29 @@
+<%= render partial: 'notices/shared/cover', :locals => {notice: notice} %>
 <%= render partial: 'notices/shared/logo' %>
-<%= render partial: 'notices/shared/date', :locals => {notice: notice} %>
 <%= render partial: 'notices/shared/address', :locals => {notice: notice} %>
+<br><br><br><br><br>
+
+<%= render partial: 'notices/shared/date', :locals => {notice: notice} %>
 
 
-<table class="total_table" border="0">
-  <tbody>
-    <tr>
-      <td><h3>REMINDER: Your Open Enrollment Period Ends Soon</h3></td>
-    </tr>
-  </tbody>
-</table>
 
 <table class="total_table" border="0">
   <tbody>
     <tr>
-      <td>Dear <%= notice.primary_fullname %>:  </td>
+      <td>Dear <%= notice.primary_fullname %>  </td>
     </tr>
   </tbody>
 </table>
 
 <%= render partial: 'notices/shared/paragraph', :locals =>
-  { content: "This is a reminder that you have until #{notice.plan_year.open_enrollment_end_on} to enroll into health coverage offered by #{notice.employer_name} through #{Settings.site.short_name}"} %>
+  { content: "This is a reminder that you have until #{notice.plan_year.open_enrollment_end_on} to enroll into health coverage offered by #{notice.employer_name} through the #{site_short_name}."} %>
 
 <%= render partial: 'notices/shared/paragraph', :locals =>
   { content: "<strong>To See Plan Options and Enroll:</strong>"} %>
 
-<%= render partial: 'notices/shared/unordered_list', locals: {
+<%= render partial: 'notices/shared/ordered_list', locals: {
   previous: "To complete the enrollment process, please complete the following steps:",
-  listings: ["Go to <a href='https://enroll.mhc.hbxshop.org'>MAhealthconnector.org</a>",
+  listings: ["Go to <a href= ""#{site_main_web_address}"">#{site_website_name}</a>",
```

Comment:
Use the href link with https:// , in that way it should work in Mozilla Firefox as well.
So please define an attribute with https:// and use a helper class ,to  call that attribute.